server/routers/transactions.router.js

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'express';
+
+const transactionRouter = Router();
+
+transactionRouter.get('/' /*to do auth, getTransactions*/);
```

Comment:
funkcja teraz nazywa się inaczej chyba? getOwnerTransactions ?

Reply:
To są funkcje z controllera, których jeszcze nie ma, tamte były w service